features/admin.flow-extensions.v1/components/flow-context-tree/flow-context-tree.scss

```
@@ -0,0 +1,193 @@
+ * under the License.
+ */
+
+// EXPOSE moved from orange to green; orange is now reserved for the brand
+// (selected-row highlight). MODIFY stays blue.
+$expose-color: #2E7D32;
+$expose-faint: #E8F5E9;
+$modify-color: #1565C0;
+$modify-faint: #E3F2FD;
+$danger-color: #C0392B;
+
+$row-selected-bg: #FFF4EC;
```

Stylelint (17.12.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)